openmetadata-ui/src/main/resources/ui/src/pages/tour-page/TourPage.component.tsx

```diff
@@ -179,6 +179,7 @@ const TourPage = () => {
             columnsUpdateHandler={handleCountChange}
             createThread={handleCountChange}
             datasetFQN={mockDatasetData.datasetFQN}
+            deletePostHandler={handleCountChange}
```

Comment:
Why do we have `handleCountChange` here instead of some delete handler?

Reply:
@darth-coder00  That is a tour page and we use mock data there.